src/main/java/it/pagopa/pn/bff/config/MsClientConfig.java

```diff
@@ -28,4 +29,13 @@ SenderReadB2BApi senderReadB2BApi(PnBffConfigs cfg) {
         senderApiClient.setBasePath(cfg.getDeliveryBaseUrl());
         return new SenderReadB2BApi(senderApiClient);
     }
+
+    @Bean
+    @Primary
+    InfoPaApi infoPaApi(PnBffConfigs cfg) {
+        it.pagopa.pn.bff.generated.openapi.msclient.external_registries.ApiClient infoPaApiClient = new it.pagopa.pn.bff.generated.openapi.msclient.external_registries.ApiClient(initWebClient(ApiClient.buildWebClientBuilder()));
+        infoPaApiClient.setBasePath(cfg.getDeliveryBaseUrl());
```

Comment:
Here we should use the ext-registries url. After merge you can use the already implemented infoPaApi